fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/FabricBiomes.java

```diff
@@ -1,37 +1,22 @@
-/*
- * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 package net.fabricmc.fabric.api.biomes.v1;
 
+import net.minecraft.util.registry.RegistryKey;
 import net.minecraft.world.biome.Biome;
 
-import net.fabricmc.fabric.impl.biome.InternalBiomeData;
+import net.fabricmc.fabric.impl.biome.FabricBiomesInternal;
 
-/**
- * General API that applies to all biome sources.
- */
 public final class FabricBiomes {
 	private FabricBiomes() { }
 
-	/**
-	 * Allows players to naturally spawn in this biome.
-	 *
-	 * @param biome a biome the player should be able to spawn in
-	 */
-	public static void addSpawnBiome(Biome biome) {
-		InternalBiomeData.addSpawnBiome(biome);
+	public static RegistryKey<Biome> register(RegistryKey<Biome> key, Biome biome) {
+		return FabricBiomesInternal.register(key, biome);
```

Comment:
What on earth

Reply:
What on earth what?

Reply:
It's definitely a bit odd to have a public api method taking a biome registry ref and biome. To me it seems like an internal map of `RegistryKey<Biome>` to `Biome`

Reply:
So you'd expect an identifier instead?
I tried to mimic the `Biomes` class here, which does it similarly.
Actually, `BuiltinRegistries.BIOME` also does it like this.

Reply:
I had no idea what it did and it seems like the old equivalent of regular register method.

Probably fine

Reply:
I could also just expose it as `FabricBiomes.REGISTRY` and then expect people to do:
`Registry.register(FabricBiomes.REGISTRY, identifier, biome`.
I was kind of hoping though, that doing it this way I could discourage people saving biome instances, which can cause issues in 1.16.2.

Reply:
All Vanilla internals now deal with Registry Keys. Anyone working with biomes will probably need to keep a RegistryKey for their Biome around.
If you want to add _another_ Overload. Fine.